presto-tests/src/test/java/io/prestosql/execution/TestEventListener.java

```diff
@@ -64,7 +64,7 @@ private void setUp()
                 .setSchema("tiny")
                 .setClientInfo("{\"clientVersion\":\"testVersion\"}")
                 .build();
-        queryRunner = new DistributedQueryRunner(session, 1);
+        queryRunner = DistributedQueryRunner.builder(session).setNodeCount(1).build();
```

Comment:
same here -- is distributed still distributed if this is one node?

Reply:
I don't think it won't be distributed, but it seems this test class depends on the single node count. Let me keep 1 as the node count in this test class for now.